pkgs/development/python-modules/notifications-android-tv/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "notifications-android-tv";
+  version = "1.0.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-jrKauDkgDyEJOLl25yeGgJDHOirDIPNgQi/z0wVNRD4=";
+  };
+
+  buildInputs = with python3.pkgs; [
```

`poetry-core` is the build system.

```suggestion
    nativeBuildInputs = [
```

---

This didn't work for me originally, but I can't reproduce that now, so I must've messed up somewhere else.